release/copy_artifacts.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Copyright 2021 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euxo pipefail
+
+PROJECT="${1?First required argument is project}"
+RELEASE_BRANCH="${2?Second required argument is release branch for example 1-18}"
+RELEASE="${3?Third required argument is release for example 1}"
```

Comment:
as mentioned above i'd presonally provide at the top and in here real example of the release rather than a random number. I hope i'm not seen as being picky, maybe is just me and how i function ;)

Reply:
I've moved the default releases into gitops style files https://github.com/aws/eks-distro/pull/137/files#diff-0afcd849197deaf1cad81cc32185ee536e2470ccd3d9e9d00513f12aac862e01R1 and the pull number now gets tacked onto the end https://github.com/aws/eks-distro/pull/137/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R7